packages/static/kairos-overlay-files/files/system/oem/32_profile.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Profile extensions"
+stages:
+  initramfs:
+    - name: "Set KUBECONFIG to access k9s and kubectl"
+      files:
+        - path: /etc/profile.d/kairos.sh
+          permissions: 0644
+          owner: 0
+          group: 0
+          content: |
+            if [ -z "$KUBECONFIG" ]; then
+                if [ -e /etc/rancher/k3s/k3s.yaml ]; then
```

Comment:
I think the first time the system boots, it takes a while for k3s to create this file. This is running in the `initrafms` stage so I'm not sure if the file would be there at that point.

Reply:
wait, my mistake. This is not running at that point, it's only been created. Ignore my comment.